tools/cell_census_builder/src/cell_census_builder/release_manifest.py

```diff
@@ -0,0 +1,130 @@
+"""
+Tools to manage the release.json manifest file.
+"""
+import json
+from typing import Dict, Optional, Union, cast
+
+import s3fs
+from typing_extensions import TypedDict
+
+from .util import urlcat
+
+"""
+The release.json schema is a semi-public format, used by all end-user packages.
+"""
+
+CensusVersionName = str  # census version name, e.g., "release-99", "2022-10-01-test", etc.
+CensusLocator = TypedDict(
+    "CensusLocator",
+    {
+        "uri": str,  # resource URI
+        "s3_region": Optional[str],  # if an S3 URI, has optional region
+    },
+)
+CensusVersionDescription = TypedDict(
+    "CensusVersionDescription",
+    {
+        "release_date": Optional[str],  # date of release, optional
+        "release_build": str,  # date of build
+        "soma": CensusLocator,  # SOMA objects locator
+        "h5ads": CensusLocator,  # source H5ADs locator
+    },
+)
+CensusDirectory = Dict[CensusVersionName, Union[CensusVersionName, CensusVersionDescription]]
```

Comment:
`CensusReleaseManifest`? "directory" sounds like a file system concept when I see this type in use.

Reply:
it is cloned from the end-user package.  Directory was intended to be used in the normal English manner, not a file system.  Will change